docs/Policies/outages-and-calendar-policy.mdx

```diff
@@ -2,7 +2,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # Outages and Calendar Policy for the Technology Group
 
-##### Last updated October 17, 2022
+##### Last updated January 26, 2023
```

Comment:
This is not valuable now that we have git. We can just git blame to see when it was last updated. I just converted this over from another doc on google drive.